h/__init__.py

```diff
@@ -24,7 +29,4 @@ def includeme(config):
     if config.registry.feature('notification'):
         config.include('h.notification')
 
-    config.include('pyramid_jinja2')
-    config.add_jinja2_renderer('.js')
-    config.add_jinja2_renderer('.txt')
-    config.add_jinja2_renderer('.html')
+    config.include('h.claim')
```

Comment:
I would add this to the .ini files after h.account rather than include it here. You can just reset this whole file. I like that you've kept this in its own package, but it depends on h.account and h/__init__.py doesn't include that.